libraries/suggestion-input-view/src/main/java/com/trendyol/suggestioninputview/SuggestionInputItemViewState.kt

```diff
@@ -20,14 +20,19 @@ data class SuggestionInputItemViewState(
     @Dimension val textSize: Float,
     @Dimension val horizontalPadding: Float,
     @Dimension val verticalPadding: Float,
-    @Dimension val minWidth: Float
+    @Dimension val minWidth: Float,
+    val isEnable: Boolean,
+    val disabledItemBackground: Drawable?
```

Comment:
```suggestion
    val disabledItemBackgroundDrawable: Drawable?
```
We can add the "drawable" to end of field name to specify the type of field